project/backend/project-manager/docker-compose.yml

```
@@ -0,0 +1,43 @@
+      - "4318:4318"
+    depends_on:
+      - zipkin
+
+  zipkin:
+    image: openzipkin/zipkin:latest
+    container_name: zipkin
+    ports:
+      - "9411:9411"
+
+  prometheus:
+   image: prom/prometheus
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 4 but found 3

(indentation)